include/openmc/cell.h

```diff
@@ -108,6 +154,12 @@ class Cell {
   //! Get the BoundingBox for this cell.
   virtual BoundingBox bounding_box() const = 0;
 
+  //! Get a vector of surfaces in the cell
+  virtual vector<int32_t> surfaces() const = 0;
```

Comment:
Similar to what I suggested for `is_simple`, you could have a default implementation that returns an empty vector